### DISCUSSION TITLE
What's New

### DISCUSSION THREAD

**Discussion by kfonda** [AT0myks/reolink-fw-archive#47] [Ideas]
Can you add the ability to get the 'what's new'/'changelog' info for the different firmware in the 'download.py' script? Basically the same info that you add to README.md, just create the readme file in the 'reolink-archive' folder when the download script runs.

**Comment by AT0myks** [comment]
Yes, that sounds pretty easy to add. I won't do it right now but I'll try to think about it next time changes are made to the script.
